numba_cuda/numba/cuda/cudadrv/mappings.py

```diff
@@ -0,0 +1,24 @@
+from numba import config
+from cuda import cuda
```

Comment:
I just noticed from the (failing) docs build that this requires CUDA Python to be installed - it needs to go inside the `if config.CUDA_USE_NVIDIA_BINDING` below.